make_queue/views/reservation/reservation.py

```diff
@@ -29,10 +30,23 @@ def get_error_message(self, form, reservation):
         """
         if not reservation.is_within_allowed_period_for_reservation() and not (
                 reservation.special or reservation.event):
-            return f"Reservasjoner kan bare lages {reservation.reservation_future_limit_days} dager frem i tid"
+            """Translation: Reservasjoner kan bare lages {:} dager frem i tid"""
+            return _("Reservations can only be made {:} days ahead of time".format(
+                reservation.reservation_future_limit_days));
```

Comment:
The closing parenthesis of `_()` should be after the end of the translation string, not after the call to `format()`. Also, the colon in `{:}` and the semicolon at the end of the statement are not needed 😋

I would suggest one of these two alternatives:
* `_("Reservations can only be made {} days ahead of time").format(reservation.reservation_future_limit_days)`
* `_("Reservations can only be made {num_days} days ahead of time").format(num_days=reservation.reservation_future_limit_days)`

The last one follows a nice convention of naming the placeholder variable(s), which helps the translators know what the type of the value is, though it's probably kind of obvious in this case.

Reply:
Even better, we can [pluralize](https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#pluralization) like this:
```python
from django.utils.translation import ngettext

num_days = reservation.reservation_future_limit_days
return ngettext(
    'Reservations can only be made {num_days} day ahead of time',
    'Reservations can only be made {num_days} days ahead of time',
    num_days
).format(num_days=num_days)
```